src/attributes.jl

```diff
@@ -285,21 +285,23 @@ end
 # examples, see ConstraintSet and ConstraintFunction. set_fallback_error should
 # not be overloaded by users of MOI.
 function set_fallback_error(model::ModelLike,
```

Comment:
This name could be misunderstood as setting a property of the model call the "fallback error". Maybe `throw_set_error_fallback`?